packages/application/components/organisms/ThankYou/index.tsx

```diff
@@ -45,21 +40,36 @@ const ThankYou: NextPage = () => {
         </HStack>
 
         <HStack py="5">
-          <LinkButton>More Details</LinkButton>
+          <Button
+            variant="link"
+            _text={{ fontSize: 'lg', color: 'primary.900' }}
+            _hover={{ _text: { color: 'primary.800' } }}
+          >
+            More Details
+          </Button>
         </HStack>
 
         <HStack py="5">
-          <ButtonPrimary large width={250}>
+          <Button
+            w={250}
+            py={7}
+            fontSize="2xl"
+          >
             Continue helping
-          </ButtonPrimary>
+          </Button>
         </HStack>
       </Box>
 
       <HStack py="8">
-        <LinkButton>Go back to Home</LinkButton>
+        <Button
+          variant="link"
+          _text={{ fontSize: 'lg', color: 'primary.900' }}
+          _hover={{ _text: { color: 'primary.800' } }}
```

Comment:
@lucianobracco-beon-studio these styles seem to be already configured in the lightTheme. However, if I remove them it uses secondary color scheme. Can you have a look?

Reply:
Oh because you have to set the colorScheme which is the prop that takes the group name of colors (for example: "primary", "secondary" etc). On the landing I have a theme somewhat configured, maybe that one could work here

Reply:
I'll iterate this in follow-up PR